solr/core/src/test/org/apache/solr/response/transform/TestChildDocTransformerHierarchy.java

```diff
@@ -124,10 +124,11 @@ public void testParentFilterLimitJSON() throws Exception {
 
     assertJQ(req("q", "type_s:donut",
         "sort", "id asc",
-        "fl", "id, type_s, toppings, _nest_path_, [child limit=1]",
+        "fl", "id, type_s, lonely, lonelyGrandChild, test_s, test2_s, _nest_path_, [child limit=1]",
```

Comment:
To my point I wrote in JIRA:  It's sad that when I see this I have no idea if it's right/wrong without having to go look at indexSampleData then think about it.  No?  (this isn't a critique of you in particular; lots of tests including some I've written look like the current tests here).   imagine one doc with some nested docs, all of which only have their ID.  Since they only have their ID, it's not a lot of literal text in JSON.  The BeforeClass unmatched docs cold use negative IDs to easily know who's who.  Any way if you would rather leave this as a "TODO" for another day then I understand.

Reply:
Leaving this as a TODO for another day sounds like a decent option.